5.3/5.3.4/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 5.3.4
+Напишите функцию checkSpam(str), возвращающую true, если str содержит 'viagra' или 'XXX', а иначе false.
+Функция должна быть нечувствительна к регистру:
+checkSpam('buy ViAgRA now') == true
+checkSpam('free xxxxx') == true
+checkSpam("innocent rabbit") == false
+<script>
+	function checkSpam(str) {
+		return (str.toLowerCase().includes('viagra') || str.toLowerCase().includes('XXX'.toLowerCase()));
```

Comment:
you can just write `str.toLowerCase().includes('xxx')` instead of `str.toLowerCase().includes('XXX'.toLowerCase()))`